.clang-format

```diff
@@ -0,0 +1,26 @@
+BasedOnStyle: Google
+UseTab: Never
+IndentWidth: 2
+ColumnLimit: 100
```

Comment:
Should we set ColumnLimit to 80 to be consistent with our ruff formating? https://github.com/vllm-project/vllm/blob/main/pyproject.toml#L13-L44

Reply:
This is a good point, let's standardize on 80